classes/Cart.php

```diff
@@ -1293,6 +1292,11 @@ public function containsProduct($id_product, $id_product_attribute = 0, $id_cust
      * @param int $id_product Product ID
      * @param int $id_product_attribute Attribute ID if needed
      * @param string $operator Indicate if quantity must be increased or decreased
+     * @param mixed $id_customization
```

Comment:
Shouldn't it be `int` or something like `int|null`?

Reply:
Getting fixed in a new PR. We have to merge this PR soon, then we will fix all mixed. It is hard to review this PR because of all the changes. And hard to solve issues, too.